test/unit/lib/plugins/aws/package/lib/strip-null-props-from-template-resources.test.js

```diff
@@ -66,4 +77,34 @@ describe('test/unit/lib/plugins/aws/package/lib/stripNullPropsFromTemplateResour
   it('Should not affect resources without null props', async () => {
     expect(Object.keys(finalTemplate.Resources.anotherBucket.Properties).length).to.equal(1);
   });
+
+  it('Should leave Properties as is if it is an empty object', async () => {
+    expect(
+      Object.prototype.hasOwnProperty.call(
+        finalTemplate.Resources.resourceWithEmptyProperties,
+        'Properties'
+      )
+    ).to.equal(true);
+    expect(
+      Object.keys(finalTemplate.Resources.resourceWithEmptyProperties.Properties).length
+    ).to.equal(0);
+  });
+
+  it('Should strip out Properties if it is defined with null value', async () => {
```

Comment:
Let's add just this test. Rest is obvious and just increases maintenance cost

Reply:
done